code/modules/admin/tickets/adminticketsverbs.dm

```diff
@@ -0,0 +1,39 @@
+//Verbs
```

Comment:
Here too

Reply:
This was addressed in the latest commit.